src/main/java/games/strategy/engine/framework/ui/background/WaitWindow.java

```diff
@@ -32,29 +41,27 @@ public void run() {
       }
     };
 
-    synchronized (m_mutex) {
-      if (m_timer != null) {
-        m_timer.schedule(task, 15, 15);
+    synchronized (mutex) {
+      if (timer != null) {
+        timer.schedule(task, 15, 15);
       }
     }
   }
 
+  /**
+   * Hides the wait window.
+   */
   public void doneWait() {
-    synchronized (m_mutex) {
-      if (m_timer != null) {
-        m_timer.cancel();
-        m_timer = null;
+    synchronized (mutex) {
+      if (timer != null) {
+        timer.cancel();
+        timer = null;
```

Comment:
The changes look good to me, but this code seems stupid to me...
Instead of repushing the frameto the front every 15 milliseconds, we could probably just use the setAlwaysOnTop() function